crates/autopilot/src/run.rs

```diff
@@ -572,17 +569,42 @@ async fn shadow_mode(args: Arguments) -> ! {
     let liveness = Arc::new(Liveness::new(args.max_auction_age));
     shared::metrics::serve_metrics(liveness.clone(), args.metrics_address);
 
+    let ethrpc = ethrpc(&args.shared.node_url).await;
```

Comment:
Potentially we could have a common function to set this up for normal run and for shadow, wdyt?